package.json

```diff
@@ -97,6 +97,11 @@
           "default": true,
           "description": "When set to true, dart/textDocument/publishClosingLabels notifications will be sent with information to render editor closing labels."
         },
+        "flutter.lsp.initialization.closingLabelPrefix": {
```

Comment:
Please change option name `flutter.lsp.initialization.closingLabelPrefix` to `flutter.closingLabelPrefix`, the `lsp.initialization` prefix options are send to lsp as initialization params.

Reply:
got it